Tickets/master/cpr 234 script updates

melissa opened the pull request.

underscorgan commented on lib/vanagon/component/source/http.rb:
```diff
@@ -13,7 +13,7 @@ class Http
 
         # Extensions for files we aren't going to unpack during the build
-        NON_ARCHIVE_EXTENSIONS = '.gem', '.ru', '.txt', '.conf', '.ini', '.gpg', '.rb', '.sh', '.csh'
+        NON_ARCHIVE_EXTENSIONS = '.gem', '.ru', '.txt', '.conf', '.ini', '.gpg', '.rb', '.sh', '.csh', '.xml'
```
This shouldn't be included.

underscorgan commented on templates/deb/control.erb:
```diff
@@ -15,7 +15,7 @@ Conflicts: <%= get_replaces.map { |replace| "#{replace.replacement} #{replace.ve
 Provides: <%= get_provides.map { |prov| prov.provide }.join(", ") %>
 Description: <%= @description.lines.first.chomp %>
- <%= @description.lines[1..-1].join("\n\s") -%>
+ <%= @description.lines[1..-1].join("\s") -%>
```
This shouldn't be included

melissa committed:
(maint) change script to scripts

The variable name should reflect the reality of the variable. There very
well may be multiple scripts

melissa committed:
(maint) we really should be including the preinst

Otherwise, we don't install the preinst file for building debian
packages. That could get very confusing later on.
